api/views/v1/tasks.py

```diff
@@ -200,6 +231,14 @@ def updateTask(taskData):
         except Task.DoesNotExist:
             return None, status.HTTP_404_NOT_FOUND
 
+        # Check if userID is in the project the task belongs to
+        userID = decodeApiKey(authorizationToken).get("userID")
+        project = Project.objects.get(id=task["projectID"])
+        if ObjectId(userID) not in project["userIDs"]:
```

Comment:
```suggestion
        project = Project.objects.get(id=task["projectID"])
        if userID not in project["userIDs"]:
```
Is this casting necessary? I've seen other instances where you iterate through project["userIDs"] and don't do this as well.

Reply:
Yes it is needed (just tested it again). When writing to the database, the mongo serializer automatically converts strings to ObjectId. However when reading from the database in this instance (`Project.objects.get(id=task["projectID"])`) without using the serializer, projectIDs are of type ObjectID which means to compare to them, I have to typecast to an ObjectID.

Reply:
good to know